app/frontend/src/features/profile/edit/ProfileForm.tsx

```diff
@@ -113,12 +145,114 @@ export default function EditProfileForm() {
+                      label={ACCEPTING}
+                    />
+                    <FormControlLabel
+                      value={HostingStatus.HOSTING_STATUS_MAYBE}
+                      control={<Radio />}
+                      label={MAYBE_ACCEPTING}
+                    />
+                    <FormControlLabel
+                      value={HostingStatus.HOSTING_STATUS_CANT_HOST}
+                      control={<Radio />}
+                      label={NOT_ACCEPTING}
+                    />
+                  </RadioGroup>
+                </>
+              )}
+            />
+            <Controller
+              control={control}
+              defaultValue={user.meetupStatus}
+              name="meetupStatus"
+              render={({ onChange, value }) => (
+                <>
+                  <Typography variant="h2">{HOSTING_STATUS}</Typography>
+                  <RadioGroup
+                    row
+                    aria-label="meetup status"
+                    name="meetupStatus"
+                    value={value}
+                    onChange={(event) => onChange(Number(event.target.value))}
+                  >
+                    <FormControlLabel
+                      value={MeetupStatus.MEETUP_STATUS_WANTS_TO_MEETUP}
+                      control={<Radio />}
+                      label={MEETUP}
+                    />
+                    <FormControlLabel
+                      value={MeetupStatus.MEETUP_STATUS_OPEN_TO_MEETUP}
+                      control={<Radio />}
+                      label={MAYBE_MEETUP}
+                    />
+                    <FormControlLabel
+                      value={MeetupStatus.MEETUP_STATUS_DOES_NOT_WANT_TO_MEETUP}
+                      control={<Radio />}
+                      label={NO_MEETUP}
+                    />
+                  </RadioGroup>
+                </>
+              )}
+            />
+            <Controller
+              control={control}
               defaultValue={user.gender}
-              inputRef={register}
-              className={classes.field}
+              name="gender"
+              render={({ onChange, value }) => (
+                <>
+                  <Typography variant="h2">{GENDER}</Typography>
+                  <RadioGroup
+                    row
+                    aria-label="gender"
+                    name="gender"
+                    value={value}
+                    onChange={(_, value) => onChange(value)}
+                  >
+                    <FormControlLabel
+                      value={FEMALE_FORM_VALUE}
+                      control={<Radio />}
+                      label={FEMALE}
+                    />
+                    <FormControlLabel
+                      value={MALE_FORM_VALUE}
+                      control={<Radio />}
+                      label={MALE}
+                    />
+                    <FormControlLabel
+                      value={OTHER_FORM_VALUE}
```

Comment:
This causes any freeform value to be set to "other". Is this intentional? I thought we want to let people to set their gender to whatever they want post registration.

Reply:
In any case, any references to gender should refer to "non-binary" rather than "other"

Reply:
The radio button doesn't actually have a label on it, it's blank. I just needed a value for the form, but it's not the label that shows.